tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_fixture.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\MethodCall\ConvertRenderTemplateShortNotationToBundleSyntaxRector\Fixture;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class BaseController extends Controller
```

Comment:
You could rename the fixture file `shortToBundleName.php.inc` and controller to `ShortToBundleName` to make clear what this fixture is changing, maybe you can find better naming